modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.scss

```diff
@@ -0,0 +1,27 @@
+.quoted-summary-container {
+	.quoted-summary-content {
+		margin-right: 200px;
+	}
+
+	.quoted-summary-title::after {
+		background-color: var(--primary);
```

Comment:
I believe that we have this color on Dialect.

Reply:
I really need to put it this way because of the "::after" selector.